applications/portfoliomanager/src/portfoliomanager/server.py

```
@@ -368,159 +362,63 @@ async def get_current_predictions() -> pl.DataFrame:
-            "Failed to fetch prior portfolio from datamanager",
-            status_code=prior_portfolio_response.status_code,
-        )
-        return empty_portfolio
-
-    # Handle empty or invalid JSON response
-    response_text = prior_portfolio_response.text.strip()
-    if not response_text or response_text == "[]":
-        logger.info("Prior portfolio response is empty")
-        return empty_portfolio
-
+def get_prior_portfolio_tickers() -> list[str]:  # noqa: PLR0911
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLR0911`)

Remove unused `noqa` directive

(RUF100)